packages/inter-protocol/src/auction/scheduleMath.js

```diff
@@ -24,25 +24,48 @@ export const computeRoundTiming = (params, baseTime) => {
   /** @type {NatValue} */
   const lowestRate = params.getLowestRate();
 
+  const noNextAuction = msg => {
+    console.error(assert.error(msg));
+    return undefined;
```

Comment:
let's use `null` to convey explicitly that there is no auction. `undefined` is returned if there is no return value.